mmdeploy/mmcv/cnn/conv2d_adaptive_padding.py

```diff
@@ -1,86 +0,0 @@
-# Copyright (c) OpenMMLab. All rights reserved.
-import math
-
-import torch
-import torch.nn.functional as F
-
-from mmdeploy.core import FUNCTION_REWRITER
-from mmdeploy.utils import Backend, is_dynamic_batch, is_dynamic_shape
-
-
-def compute_padding(input_size, kernel_size, stride, dilation):
-    """Compute padding."""
-
-    input_h, input_w = input_size
-    kernel_h, kernel_w = kernel_size
-    stride_h, stride_w = stride
-    dilation_h, dilation_w = dilation
-    output_h = math.ceil(input_h / stride_h)
-    output_w = math.ceil(input_w / stride_w)
-    pad_h = max(
-        (output_h - 1) * stride_h + (kernel_h - 1) * dilation_h + 1 - input_h,
-        0)
-    pad_w = max(
-        (output_w - 1) * stride_w + (kernel_w - 1) * dilation_w + 1 - input_w,
-        0)
-    if pad_w > 0 or pad_h > 0:
-        padded = [
-            pad_w // 2, pad_w - pad_w // 2, pad_h // 2, pad_h - pad_h // 2
-        ]
-    else:
-        padded = None
-    return padded
-
-
-class AdaptivePadOp(torch.autograd.Function):
-    """Dummy adaptive pad op."""
-
-    @staticmethod
-    def forward(ctx, x, padded):
-        if padded is not None:
-            x = F.pad(x, padded)
-        return x
-
-    @staticmethod
-    def symbolic(g, x, padded):
-        if padded is None:
-            return g.op('Identity', x)
-        padded = g.op(
-            'Constant', value_t=torch.tensor(padded, dtype=torch.int64))
-        constant_value = g.op(
-            'Constant', value_t=torch.tensor(0, dtype=torch.int64))
-        return g.op(
-            'Pad', x, padded, constant_value, mode_s='constant', outputs=1)
-
-
-@FUNCTION_REWRITER.register_rewriter(
-    func_name='mmcv.cnn.bricks.conv2d_adaptive_padding. \
-        Conv2dAdaptivePadding.forward',
-    backend=Backend.TENSORRT.value)
-def conv2d_adaptive_padding__forward__tensorrt(ctx, self, x):
```

Comment:
why do we remove this rewriting?

Reply:
`'mmcv.cnn.bricks.conv2d_adaptive_padding. \
        Conv2dAdaptivePadding.forward'` will not map to any function. This rewriter has never been enabled.

Reply:
It is added by @SsTtOoNnEe for deployment of mmcls::efficientnet for torch>=1.10
https://github.com/open-mmlab/mmcv/blob/2478cb6f840206eb0d78cd27320d9068be483860/mmcv/cnn/bricks/conv2d_adaptive_padding.py#L46

Reply:
The rewriter has never been enabled. The func_name string is WRONG.

Reply:
then we could fix the `func_name` instead of removing the rewriting.

Reply:
Enable the rewriter would failed the export of efficientnet